.github/workflows/tests.yml

```diff
@@ -164,7 +164,7 @@ jobs:
 
           - description: Verify distribution build
             name: test-build-dist
-            run: npm run build:dist
+            run: npm run build:release
```

Comment:
Here's an example of the slight confusion in naming - the description here referring to "distribution", the  task referring to "release".

Reply:
Yeah this needs some work doesn't it

Ideally we have a single consolidated distribution via `npm publish` rather than another separate one for GitHub. Seems a shame that `npm install govuk-frontend` doesn't include those fab precompiled files

I didn't want to go that far in this PR though so I'm open to better naming/directory conventions

Reply:
We're going to [keep `./dist` at project root](https://github.com/alphagov/govuk-frontend/pull/3491#discussion_r1173953748) for now